apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/HHAClaimTransformerV2Test.java

```diff
@@ -274,7 +274,9 @@ public void shouldHaveDischargeStatusSupInfo() {
                     "Discharge Status")),
             // Code
             new Coding(
-                "https://bluebutton.cms.gov/resources/variables/ptnt_dschrg_stus_cd", "1", null));
+                "https://bluebutton.cms.gov/resources/variables/ptnt_dschrg_stus_cd",
+                "30",
```

Comment:
I'm seeing how these mapping errors occur; we're just using a hardcoded value that isn't pulled from the source data field in the tests. Nothing to do for this PR, but it seems like these tests could use a better expected value by pulling from the expected mapping location of the input, so we make sure ccw data input -> output path/value is correct.